src/libraries/System.Memory/src/System/ThrowHelper.cs

```diff
@@ -121,6 +121,6 @@ internal enum ExceptionArgument
         culture,
         manager,
         count,
-        writer,
+        writer
```

Comment:
Why?The comma is unuseful it's the last value.

Reply:
Well, this is a cosmetic change unrelated to the PR, so we generally try to avoid such changes (or limit them).

Also, the comma was left there intentionally so that when new entries are added to the list of enums, only the new enum values show up in the diff and the author doesn't need to add an extra comma at the end of the previous value. Using this pattern is just personal preference (not enforced) but it shows up in a quite few places within the code base.

In this particular case, see https://github.com/dotnet/corefx/pull/37661#discussion_r294842733

Hence, to me, we should undo this change. If we want to make such cosmetic changes, consider doing it across the whole repo in a separate PR, but like I said, it might not be helpful.

If you search for this regex in all `*.cs` files, you will see ~5k hits (many related to to enums):
`,\r\n[\s]*\}`

Here are some examples:
https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/System.Linq/src/System/Linq/ThrowHelper.cs#L79-L80https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.cs#L685-L686
https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/CSharpBinderFlags.cs#L65-L66
https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/System.Collections.Immutable/tests/ImmutableListTest.cs#L25-L26
https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/System.ComponentModel.Primitives/src/System/ComponentModel/RefreshProperties.cs#L26-L27
https://github.com/dotnet/runtime/blob/9c130e4863dab0512713e27a8ee44f303af44ebf/src/libraries/Microsoft.Win32.Registry/tests/TestData.cs#L117-L118

Reply:
Well, I understood the reason now, I thought was an error, but actually help review TIL...I'll revert.